.github/workflows/check-and-manage-shavar-files-updates.yml

```diff
@@ -0,0 +1,116 @@
+name: Run script on shavar-creation-lists
+on:
+  schedule:
+    # Weekly on Monday at 00
+    - cron: '0 0 * * 1'
+
+permissions:
+  contents: 'read'
+  id-token: 'write'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        persist-credentials: false 
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - uses: actions/checkout@v3
+      with:
+        repository: 'irios3/shavar-list-creation'
```

Comment:
I'm using my fork becuase I had to modify the list2safebrowsing.py script to fix and issue with and how it manages some branches' names